### ISSUE TITLE
Feature Request: Allow customizing HTML Title

### ISSUE THREAD

**Issue by bb** [axllent/mailpit#154]
Currently the HTML Title is always "Mailpit". This is fine as long as only one instance is used.

With multiple environments, e.g. local and staging, I'm running multiple instances so it would be great if I could name them "Mailpit Local" and "Mailpit Staging".

Adding e.g. `MP_UI_TITLE` and `--ui-title` with default "Mailpit" would be great.

What do you think?

**Comment by axllent** [comment]
Interesting suggestion @bb. When you say HTML title, I assume you mean the `<title>` in the `<head>` (ie: the tab name), and not the "Mailpit" title (headling/link) at the top of the page?

When I started Mailpit I made an intentional effort to keep the CLI flags/options to a minimum, but that grew and grew as popularity picked up and feature requests came in. I still have to make a conscious effort to keep things as simple as possible where avoidable so I'd rather not add new flags if they can be achieved in a different manner.

Thinking aloud here ~ are your Mailpit servers running on the same hostname/ip? What I mean is, what if I simply used the current hostname (that you were accessing Mailpit with) in the `<head>` `<title>`, so for example `Mailpit - staging.example.com`? Would that work for you?
